### PR TITLE
Update location-verification-API-Readiness-Checklist.md

### PR DESCRIPTION
**Pull request by jlurien**
#### What type of PR is this?

* subproject management

#### What this PR does / why we need it:

Updates API readiness checklist towards the milestones. RM team can check status in this PR

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

**Review by bigludo7 — APPROVED** [review]
LGTM but we can probably already add the links for API definition and API documentation  no?

**Comment by jlurien** [comment]
> LGTM but we can probably already add the links for API definition and API documentation no?

Yes, I just drafted the PR to take it over from @jgarciahospital :)

**Review by jgarciahospital — APPROVED** [review]

**Review by bigludo7 — DISMISSED** [review]
I'm not sure we need links for lines 2 & 3 (see [here](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API-Readiness-Checklist.md)) - I'm fine you add these (and approve) but for consistence with other API I will probably prefer to not have them.

File: documentation/API_documentation/location-verification-API-Readiness-Checklist.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# API Readiness Checklist
+
+Checklist for location-verification v1.0.0 in rx.y.
+
+| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
+|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
+|  1 | API definition                               |   M   |         M         |    M    |    M   |Y| [OAS](https://github.com/camaraproject/DeviceLocation/blob/main/code/API_definitions/location-verification.yaml) |
```
Comment (hdamker) [inline documentation/API_documentation/location-verification-API-Readiness-Checklist.md:7]: Relative links would work here (and point to the released files):

```suggestion
|  1 | API definition                               |   M   |         M         |    M    |    M   |Y| [OAS](/code/API_definitions/location-verification.yaml) |
```

File: documentation/API_documentation/location-verification-API-Readiness-Checklist.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |Y| [r0.4.0-rc.1](https://github.com/camaraproject/Commonalities/tree/r0.4.0-rc.1)     |
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |Y| [r0.2.0-rc.1](https://github.com/camaraproject/IdentityAndConsentManagement/tree/v0.2.0-rc.1)     |
+|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |Y| v1.0.0 |
+|  5 | API documentation                            |   M   |         M         |    M    |    M   |Y|      |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |TBC| link |
+|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |Y| [Gherkin](https://github.com/camaraproject/DeviceLocation/blob/main/code/Test_definitions/location-verification.feature) |
```
Comment (hdamker) [inline documentation/API_documentation/location-verification-API-Readiness-Checklist.md:13]:
```suggestion
|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |Y| [Gherkin](/code/Test_definitions/location-verification.feature) |
```

File: documentation/API_documentation/location-verification-API-Readiness-Checklist.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |Y | [Gherkin](https://github.com/camaraproject/DeviceLocation/blob/main/code/Test_definitions/location-verification.feature) |
```
Comment (hdamker) [inline documentation/API_documentation/location-verification-API-Readiness-Checklist.md:14]:
```suggestion
|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |Y | [Gherkin](/code/Test_definitions/location-verification.feature) |
```

**Review by bigludo7 — APPROVED** [review]

**Comment by hdamker** [comment]
It doesn't matter really, but if `./CHANGELOG.md` would be a link, it wouldn't work. Correct would be `/CHANGELOG.md`

**Pull request merged**